src/test/server/write_buffer_test.cpp

```diff
@@ -0,0 +1,75 @@
+#include <boost/asio.hpp>
+
+#include "base_test.hpp"
+#include "gtest/gtest.h"
+
+#include "mock_socket.hpp"
+
+#include "server/write_buffer.hpp"
+
+namespace opossum {
+
+class WriteBufferTest : public BaseTest {
```

Comment:
Does this have something that triggers flush if necessary without an explicit flush?

Reply:
This is covered by this test: https://github.com/hyrise/hyrise/blob/aa84e212c6d8d0be24639daa1f6ce8a9b5a2b77c/src/test/server/write_buffer_test.cpp#L68-L73

Reply:
But here you call `_write_buffer->flush(); ` explicitly before you test something.

Reply:
Got your point. Gonna add an explicit EXPECT before `flush`.